lib/pdf/core/page.rb

```diff
@@ -167,6 +173,9 @@ def init_new_page(options)
         @dictionary = document.ref(:Type        => :Page,
                                    :Parent      => document.state.store.pages,
                                    :MediaBox    => dimensions,
+                                   :CropBox     => dimensions,
+                                   :BleedBox    => dimensions,
```

Comment:
Ok, thanks. My first time digging into PDF internals, so I'll take whatever advice you've got.  All the stuff I submitted is working, but I've got a lot to learn about this spec before I feel comfortable saying how it should be.